components/typography/config.less

```diff
@@ -1,12 +1,13 @@
 @line-height-base: 1.4;
 
-@font-size-base: 14px;
-@font-size-xsmall: 11px;
-@font-size-small: 12px;
-@font-size-large: 16px;
+@font-size-base: 15px;
+@font-size-xsmall: 12px;
+@font-size-small: 13px;
+@font-size-large: 17px;
+
 // Open Sans cap height
 @cap-height: 0.7727272727;
 
-@font-family-sans-serif: "open-sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
-@font-family-serif: "ff-tisa-sans-web-pro", serif;
+@font-family-sans-serif: "proxima nova", "Helvetica Neue", Helvetica, Arial, sans-serif;
+@font-family-serif: "effra", sans-serif;
```

Comment:
I guess since you are changing variable names, this would be a breaking change too. Would have to do a major version bump. You know these changes are also going to be applied to checkout too, right?